aspen/simplates/renderers/json_dump.py

```diff
@@ -8,7 +8,7 @@
 from __future__ import unicode_literals
 
 from . import Renderer, Factory
-from .. import json
+from ... import json
```

Comment:
We still have `aspen.json`? Weren't we going to move that to `aspen.simplates.json`?